### PR TITLE
Materialized view setup

### PR DESCRIPTION
**Pull request by deanott**

### CODE REVIEW

File: pipeline/db/db.go [hunk 1]
```diff
@@ -717,15 +717,15 @@ func QueryAll(
...
+	// 	"FULL OUTER JOIN app_packages pkg  ON (a.id = pkg.id) "
 
 	queryStr = "SELECT " +
 		"id, title, summary, description, store_url, price, free, rating, " +
 		"num_reviews, genre, family_genre, min_installs, max_installs, updated, " +
 		"android_ver, content_rating, recent_changes, app, store, region, " +
 		"version, icon, analyzed, email, name, store_site, site, hosts, permissions, " +
 		"packages " +
-		"FROM  all_apps_data " +
+		"FROM  matview.apps_play_data " +
```
Comment (AdamSlack) [inline pipeline/db/db.go]: matview -> mat_view

File: pipeline/db/db.go [hunk 2]
```diff
@@ -847,6 +855,7 @@ func QueryAll(
 	util.Log.Debug("Examining rows")
 
 	result := []AppVersion{}
+	util.Log.Debug("wtf?")
```
Comment (AdamSlack) [inline pipeline/db/db.go]: What is this actually saying? :P

File: pipeline/db/init_speed_mat_views.sql [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	FULL OUTER join app_perms p on (a.id = p.id) 
+	FULL OUTER join app_packages pkg  on (a.id = pkg.id)  order by max_installs;
+
+create index apps_title_gin_trgm_idx on matview.apps_play_data USING GIN (title gin_trgm_ops);
+
+grant select on matview.apps_play_data to apiserv;
+
+
+
+commit;
```
Comment (AdamSlack) [inline pipeline/db/init_speed_mat_views.sql]: new line ending?

**Review by AdamSlack — CHANGES_REQUESTED** [review]
View comments for requested changes

File: pipeline/db/db.go (hunk 1 above)
Comment (deanott) [reply pipeline/db/db.go]: but like its already changed on the database? :L

File: pipeline/db/db.go (hunk 2 above)
Comment (deanott) [reply pipeline/db/db.go]: it will go now...

**Review by AdamSlack — APPROVED** [review]

**Pull request merged**